.github/workflows/release.yml

```
@@ -22,16 +22,16 @@ on:
         options: [ubuntu-latest, windows-latest, macos-latest]
         default: ubuntu-latest
       dry-run:
+        description: Dry Run - Check to run the workflow without creating a release.
         required: false
-        description: If true, the release will not be created.
         default: false
         type: boolean          
 
 
 jobs:
```

Need to add a job here to conditionally get the dir path to the release notes based on the type of release

```suggestion
jobs:
```